languages/clojure/config.json

```diff
@@ -3,12 +3,16 @@
   "active": true,
   "blurb": "a JVM hosted functional programming language.",
   "version": 3,
-  "online_editor": {
-    "indent_style": "space",
-    "indent_size": 2
-  },
+  "online_editor": { "indent_style": "space", "indent_size": 2 },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "lists",
+        "uuid": "where to get this?",
```

Comment:
Any uuid generator will do - for convenience, the configlet tool can do it: https://github.com/exercism/configlet#uuid

Reply:
@porkostomus thanks! used an online one to generate, hopefully it's not duplicated.

I tried first the configlet, but unable to get it run: (i see the configlet app already under the downloads folder, but it simply reports not found. no idea)

```
192:downloads $ configlet
-bash: configlet: command not found
```